sources/Context.cpp

```diff
@@ -167,6 +167,7 @@ void Context::filterPartiallyConnectedDynamicModels() {
       if (found == modelDef.nodeConnections.end()) {
         LOG(debug, ModelPartiallyConnected, automaton.second.id);
         dynamicModels_.models.erase(automaton.second.id);
+        removeRpclFromGenerators(automaton.second.macroConnects);
```

Comment:
This should be done only if the model is a SVC with a missing connection to a node

Reply:
still not solved